src/main/resources/static/css/styles.css

```
@@ -1,35 +1,187 @@
+    }
+}
 /* Header */
 header {
-    background: linear-gradient(135deg, #667eea 0%, #764ba2 100%);
-    color: white;
+    background: transparent;
+    color: var(--base-color-white);
     padding: 40px 30px;
     text-align: center;
+
+    border-top: solid #f7f7f717;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)